pkg/cmd/openshift-sdn/cmd.go

```diff
@@ -165,6 +177,14 @@ func (sdn *OpenShiftSDN) ValidateAndParse() error {
 		return err
 	}
 
+	// If our CNI config file exists, remove it so that kubelet doesn't think
+	// we're ready yet
```

Comment:
Comment is wrong here. (This code is just initializing variables, the removal is elsewhere.)